pkg/kessel/config/config.go

```diff
@@ -0,0 +1,83 @@
+package config
+
+import (
+	"crypto/tls"
+	"time"
+)
+
+type Config struct {
+	// Endpoint specifies the server address (host:port)
+	Endpoint string `json:"endpoint" env:"KESSEL_ENDPOINT"`
+
+	// Timeout specifies the default request timeout
+	Timeout time.Duration `json:"timeout" env:"KESSEL_TIMEOUT" default:"10s"`
+
+	// UserAgent specifies the client user agent
+	UserAgent string `json:"user_agent" env:"KESSEL_USER_AGENT" default:"kessel-go-sdk"`
+
+	// UseHTTP forces HTTP instead of gRPC
+	UseHTTP bool `json:"use_http" env:"KESSEL_USE_HTTP"`
```

Comment:
Timeouts I think are only relevant to HTTP, because for gRPC deadlines should come from [service config](https://docs.google.com/document/d/1Bvjy4twwa5_xPc74OEZI2pCUgDKgfAjlw3RMBZ-UwJA/edit?tab=t.0#heading=h.5ves3wd9onxc).

This is an example of something I would expect generally: for HTTP and gRPC to use some different configurations. We may want to explicitly delineate HTTP configuration from gRPC configuration in the config object so as not to be confusing. You might reasonably think, for example, that timeout applies to gRPC when it actually doesn't.